src/routes/countryRoutes.js

```diff
@@ -0,0 +1,22 @@
+import Router from "express";
+import controllers from "../controllers";
+import Authentication from "../middlewares/authenticate";
+
+const { verifyAdmin, verifyToken } = Authentication;
+
+const { countriesController } = controllers;
+const {
+  listCountries, getCountry, deleteCountry, updateCountry
+} = countriesController;
+
+const router = Router();
+
+router.get("/get-country", getCountry);
```

Comment:
also, since we are dealing with ids her it should be `router.get("/admin/get-country/:id", getCountry);`